operator-metadata/manifests/strimziclusteroperator.configmap.yaml

```diff
@@ -1,7 +1,7 @@
 kind: ConfigMap
 apiVersion: v1
 metadata:
-  name: amq-streams-cluster-operator
```

Comment:
Do we not need amq-streams in the name here?

Reply:
I am totally fine with stripping it out if there is no need for it as it will save us time!

Reply:
we mount this ConfigMap as volume in the operator deployment, so in our CSV it's declared this way:

```yaml
- name: co-config-volume
                  configMap:
                    name: amq-streams-cluster-operator
```

so they have to match here and there. If changing in one place we have to change in the other. Why do you want to change?

Reply:
> Why do you want to change?

I am fine with either way, just noticed that it was changed from `amq-streams` -> `strimzi` in the diff. IDK why we had it as `amq-streams` for past releases

If we changed to `strimzi` in both places, it would save us from having to remember to rebrand to `amq-streams` when we backport these files

Reply:
So I am down for leaving it as 'amq-streams' or updating to `strimzi` in both places, whichever is preferred

Reply:
Now I realized it's a good catch Kyle! In the installation files we have `strimzi-cluster-operator` so if you agree I would do the same here and in the CSV.